templates/partial/_careers_navigation.html

```diff
@@ -0,0 +1,190 @@
+<div class="dropdown-window slide-animation" aria-hidden="true" aria-expanded="false">
```

Comment:
Could you fix the indentation on this file? I believe our convention is 2 spaces